zebra-network/Cargo.toml

```diff
@@ -2,7 +2,9 @@
 name = "zebra-network"
 version = "1.0.0-beta.25"
 authors = ["Zcash Foundation <zebra@zfnd.org>"]
+description = "Networking code for Zebra"
 license = "MIT OR Apache-2.0"
```

Comment:
It looks like `peer_set/set.rs` was copied from `tower-balance` as well:
https://github.com/tower-rs/tower/blob/master/tower/src/balance/p2c/service.rs

So `zebra-network` will need the same license changes as `tower-batch`:
- [ ] add the tower authors to the list of crate `authors`:
https://github.com/tower-rs/tower/blob/bf4ea948346c59a5be03563425a7d9f04aadedf2/tower/Cargo.toml#L10
- [ ] add a separate `LICENSE` file that includes the Tower copyright statement in https://github.com/tower-rs/tower/blob/master/tower/LICENSE

(The copied code is already documented.)

Reply:
@teor2345 would a file named `LICENSE-TOWER` be appropriate, as opposed to just `LICENSE`?

Reply:
This is strictly the MIT license, we could just match it to the license in the `Cargo.toml` and include as `LICENSE` or `LICENSE-MIT`; not sure if specifying `MIT` in the license field is sufficient, as many crates just include that field?

Reply:
Looks there is an /r/rust thread that discusses this explicitly but the subreddit has gone private as part of the reddit API protest, I think
https://www.reddit.com/r/rust/comments/spwo00/is_specifying_license_in_cargotoml_considered/

Reply:
One answer there basically says at least for MIT variants, the `license` field is not sufficient, so

Reply:
The copyright notices need to include both tower and ZF, I'll push a PR.

Reply:
> One answer there basically says at least for MIT variants, the `license` field is not sufficient, so

The licensing requirement from the MIT license text is:
> The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.

So no, it's not enough to modify the `license` and `authors` fields, we have to copy the LICENSE file itself.

Reply:
Fixed in commit 7632aef